packages/amplify-migration/src/index.ts

```diff
@@ -118,6 +119,43 @@ const resolveAppId = (): string => {
   return meta?.providers?.awscloudformation?.AmplifyAppId;
 };
 
+const unsupportedCategories = (): Map<string, string> => {
+  const unsupportedCategories = new Map<string, string>();
+
+  unsupportedCategories.set('geo', 'https://docs.amplify.aws/react/build-a-backend/add-aws-services/geo/');
```

Comment:
nit: can set the prefix url (https://docs.amplify.aws/react/build-a-backend/) in a variable for repeated access.